.github/pull_request_template.md

```diff
@@ -0,0 +1,75 @@
+⚠️ Click preview button above ⚠️
+
+# PR Template
+
+1. Apply relevant labels.
+1. Self-review code and PR.
+1. Request review from a team member.
+1. Wait for the code review.
+1. Take the bits that apply to your PR and delete the rest of template.
```

Comment:
Are these tasks put in order? If not, is there a sequence here that we should follow?

Reply:
Yes, they are in order.